gallery/src/components/NxRadio/NxRadioExample.tsx

```diff
@@ -29,6 +29,14 @@ export default function NxRadioExample() {
                  radioId="color-purple">
           Purple (disabled)
         </NxRadio>
+        <NxRadio name="color"
+                 value="yellow"
+                 onChange={setColor}
+                 isChecked={true}
+                 disabled={true}
+                 radioId="color-yellow">
+          Yellow (disabled + checked)
+        </NxRadio>
```

Comment:
This messes up the interaction with the rest of the radios in this example.  I think the disabled radios should be split out into a separate example.

Reply:
> This messes up the interaction with the rest of the radios in this example. I think the disabled radios should be split out into a separate example.

Separated out the disabled radios into a different GalleryExampleTile example.
[999c3bf](https://github.com/sonatype/sonatype-react-shared-components/commit/999c3bf488545310eac337be44aa71d376b710ca)